### PR TITLE
Simplified Chinese localization

### PR DESCRIPTION
**Pull request by YunjieLi**
Per chat w/ @1ec5 adding the Chinese localization.

### CODE REVIEW

**Comment by YunjieLi** [comment]
A few notes:
- didn't touch the .stringsdict since they're not needed for Chinese.
- left out two strings that look like error messages in [localizable.strings](https://github.com/mapbox/mapbox-gl-native/pull/7316/commits/aa420f3377ef03d5ed14076be7eacba8e8e845f9#diff-b21c7c76bef311a95bef11f62e91e752R4).
- Not sure what this means `"MAP_A11Y_VALUE" = "Zoom %1$dx\n%2$ld annotation(s) visible";`
- telemetry is `传感数据` according to @xinnong 
- used 你 not 您 (like tu vs. vous in French) for `you` to make it sound more friendly and less businesslike

File: platform/ios/resources/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+"CANCEL" = "取消";
+
+/* Accessibility hint */
+"COMPASS_A11Y_HINT" = "旋转地图使正北朝上";
+
+/* Accessibility label */
+"COMPASS_A11Y_LABEL" = "指南针";
+
+/* Compass abbreviation for north */
+"COMPASS_NORTH" = "N";
```
Comment (YunjieLi) [inline platform/ios/resources/zh-Hans.lproj/Localizable.strings]: I personally prefer N for brevity but no objections!

File: platform/ios/app/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,6 +1,16 @@
 {
   "images" : [
     {
+      "idiom" : "iphone",
```
Comment (1ec5) [inline platform/ios/app/Assets.xcassets/AppIcon.appiconset/Contents.json]: These changes look unrelated. If they’re needed, would you mind splitting them out into a separate PR? Thanks.

File: platform/darwin/resources/zh-Hans.lproj/Foundation.stringsdict [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>COORD_DEG_LONG</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@degrees@</string>
```
Comment (1ec5) [inline platform/darwin/resources/zh-Hans.lproj/Foundation.stringsdict]: Since Chinese doesn’t need to pluralize any strings, leave this file out so Chinese speakers don’t see English. `COORD_DEG_LONG` etc. are already defined in Localizable.strings.

File: platform/ios/resources/zh-Hans.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+"COPY_OSM" = "© OpenStreetMap";
+
+/* Instructions in Interface Builder designable; {key}, {plist file name} */
+"DESIGNABLE" = "在%2$@中将你的access token设为%1$@可在这里显示Mapbox上的地图\n\n更多说明请见";
+
+/* Setup documentation URL display string; keep as short as possible */
+"FIRST_STEPS_URL" = "mapbox.com/help/first-steps-ios-sdk";
+
+/* Accessibility hint */
+"INFO_A11Y_HINT" = "显示致谢，用户反馈及更多";
```
Comment (1ec5) [inline platform/ios/resources/zh-Hans.lproj/Localizable.strings]: Should the list items be separated by `，` or `、`?

File: platform/ios/resources/zh-Hans.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+"INFO_A11Y_LABEL" = "关于这个地图";
+
+/* Accessibility label */
+"LOGO_A11Y_LABEL" = "Mapbox";
+
+/* Accessibility label */
+"MAP_A11Y_LABEL" = "地图";
+
+/* Map accessibility value */
+"MAP_A11Y_VALUE" = "Zoom %1$dx\n%2$ld annotation(s) visible";
```
Comment (1ec5) [inline platform/ios/resources/zh-Hans.lproj/Localizable.strings]: After the user zooms the map, VoiceOver says, for example:

> Zoom 2x
> 12 annotations visible

`%1d` is a placeholder for the zoom scale (but `x` is pronounced literally in English). `\n` is a newline. `%2$ld` is a placeholder for the number of visible annotations.

If “annotations” doesn’t translate well, you can try something like “pins”, “markers”, or “map items”.

File: platform/ios/resources/zh-Hans.lproj/Localizable.stringsdict [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>MAP_A11Y_VALUE</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>Zoom %dx
```
Comment (1ec5) [inline platform/ios/resources/zh-Hans.lproj/Localizable.stringsdict]: Likewise, you can leave this file out too.

File: platform/ios/resources/zh-Hans.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,84 @@
+/* Accessibility hint */
+"ANNOTATION_A11Y_HINT" = "显示信息";
+
+/* No comment provided by engineer. */
+"API_CLIENT_400_DESC" = "The session data task failed. Original request was: %@";
```
Comment (1ec5) [inline platform/ios/resources/zh-Hans.lproj/Localizable.strings]: These error messages come from Mapbox Telemetry. In general, error messages should be translated whenever possible. However, in this case, they aren’t ever presented to the user, so it’s fine to leave these messages in English.

File: platform/ios/resources/zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (1ec5) [inline platform/ios/resources/zh-Hans.lproj/Localizable.strings]: 北 fits and is consistent with MapKit’s Chinese localizations, so let’s go with that.

**Comment by 1ec5** [comment]
This PR is targeted at the master branch. It’s probably too late to take this PR for v3.4.0, but I think we could get it into v3.4.1 as long as we rebase the branch and retarget the PR upon the release-ios-v3.4.0 branch.

File: platform/ios/app/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
Comment (YunjieLi) [inline platform/ios/app/Assets.xcassets/AppIcon.appiconset/Contents.json]: I did not make that change manually. can I just delete that line?

Comment (1ec5) [inline platform/ios/app/Assets.xcassets/AppIcon.appiconset/Contents.json]: Sure.

**Comment by YunjieLi** [comment]
@friedbunny @1ec5 just incorporated the feedbacks above. Want to take another look?

File: platform/macos/CHANGELOG.md [hunk 8]
```diff
@@ -5,6 +5,7 @@
 * Labels written in Arabic, Hebrew, and other scripts are written right-to-left. Arabic characters are correctly shaped. ([#6984](https://github.com/mapbox/mapbox-gl-native/pull/6984), [#7123](https://github.com/mapbox/mapbox-gl-native/pull/7123))
 * Improved the line wrapping behavior of point-placed labels written in Chinese, Japanese, and Yi. ([#6828](https://github.com/mapbox/mapbox-gl-native/pull/6828))
 * Fixed an issue where translucent point annotations along tile boundaries would be drawn darker than expected. ([#6832](https://github.com/mapbox/mapbox-gl-native/pull/6832))
+* Added the Simplified Chinese localization. ([#7316](https://github.com/mapbox/mapbox-gl-native/pull/7316))
```
Comment (1ec5) [inline platform/macos/CHANGELOG.md]: I just realized this PR doesn’t actually localize the macOS SDK. It could – you’d add the localization to macos.xcodeproj, reuse platform/darwin/resources/zh-Hans.lproj/Foundation.strings, and add platform/macos/resources/zh-Hans.lproj/Localizable.strings ([just 10 easy strings](https://github.com/mapbox/mapbox-gl-native/blob/3d2dc471918a38610105509a73eda39d5112ce6b/platform/macos/sdk/Base.lproj/Localizable.strings)). But we can take care of that in a separate PR. Would you mind removing this changelog entry in the meantime?

**Comment by friedbunny** [comment]
Oh, sorry, we forgot to mention [one thing](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md): because this is such a busy repo with so many different platforms, we prefix commits messages with the affected platforms. E.g., `[ios] Added super-awesome localization`.

Just something to keep in mind for next time. 👍

**Comment by YunjieLi** [comment]
Roger that @friedbunny

**Comment by 1ec5** [comment]
Opened #7370 for a Traditional Chinese localization based on this one and #7371 for a Simplified Chinese localization in the macOS SDK.
